petab/lint.py

```diff
@@ -24,13 +36,24 @@ def check_condition_df(df):
             f"Condition table has wrong index {df.index.name}."
             "expected 'conditionId'.")
 
+    assert_presence_of_empty_spaces(df.index.values, "conditionId")
+
+    for column_name in req_cols:
+        if df[column_name].dtype != np.dtype(np.float) and df[column_name].dtype != np.dtype(np.int):
+            assert_presence_of_empty_spaces(df[column_name].values, column_name)
+
 
 def check_measurement_df(df):
     req_cols = [
         "observableId", "preequilibrationConditionId", "simulationConditionId",
         "measurement", "time", "observableParameters", "noiseParameters",
         "observableTransformation"
     ]
+
+    for column_name in req_cols:
+        if df[column_name].dtype != np.dtype(np.float) and df[column_name].dtype != np.dtype(np.int):
```

Comment:
Check for common numeric supertype using numpy or Numbers.

Reply:
I used the following now:
`np.issubdtype(df[column_name].dtype, np.number)`

Reply:
Yeah, that's good